from_cpython/CMakeLists.txt

```diff
@@ -158,21 +156,7 @@ add_library(FROM_CPYTHON OBJECT ${STDMODULE_SRCS} ${STDOBJECT_SRCS} ${STDPYTHON_
 add_dependencies(FROM_CPYTHON copy_stdlib)
 
 set(STDMODULES
-    ${CMAKE_BINARY_DIR}/lib/python2.7/lib-dynload/bz2.pyston.so
```

Comment:
Just curious, why do you want to remove these lines?  It's definitely annoying that we have to duplicate all this between the setup.py and CMakeLists.txt (and same with the source files), but it means that our cmake system will end up knowing what the setup.py will do.  I think concretely the features it gets us are 1) not having to run setup.py if none of the dependencies change, and 2) properly packaging the .so's into a release tarball.

How would you feel about keeping these definitions + potentially adding any new ones from the new setup.py?  We could probably handle the .c files by using a glob_recurse, but we might have to list out all the .so files we are expecting.